scripts/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "scripts",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "seed-db": "npx ts-node src/seed-database.ts",
```

Comment:
is seed-db still used?

Reply:
yes, it populates firestore with an initial template + experiment config